app/models/service_provider_identity.rb

```diff
@@ -15,6 +15,7 @@ class ServiceProviderIdentity < ApplicationRecord
   # rubocop:enable Rails/InverseOf
 
   scope :not_deleted, -> { where(deleted_at: nil) }
+  scope :consented, -> { where.not(last_consented_at: nil) }
```

Comment:
Dropping a "requested change" as something of a formality while we re-evaluate this set of changes internally